tests/Base.php

```diff
@@ -76,7 +76,9 @@ abstract class Base extends TestCase
 
     protected const MULTIPART_RESPONSES = [
         'abc',
-        'd80e7e6999a3eb2ae0d631a96fe135a4' #
+        'd80e7e6999a3eb2ae0d631a96fe135a4',
+        '{"key": "value", "key2": "value2"}',
+        'value',
```

Comment:
A bit scared to use such simple term. TO be extra safe, lets do `stringPayloadValue` or similar

Reply:
Changed